### ISSUE TITLE
Android feature: strech screen

### ISSUE THREAD

**Issue by vladone97** [citra-emu/citra#7174]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What feature are you suggesting?

When playing in the single screen layout mode, there should be an option to stretch the 3DS screen over the enitre surface of the phone screen.

### Why would this feature be useful?

The way some games are designed makes them really easy to play in single screen landscape mode (for example DOA Dimensions or Super Street Fighter 4), but the black borders are really annoying.

**Comment by SleepingSnakezzz** [comment]
You can technically apply a custom layout to do this already, however, this will make the image look severely stretched on a lot of devices. So we'd rather not introduce such a feature outside of a GUI custom layouts editor, where it would look horribly for the vast majority of users.

**Comment by vladone97** [comment]
Yeah, I know about that feature. You can do it from the config ini file, but with that you can't swap screens. The idea here is to have the top 3DS screen stretched over the entire phone screen, since that's the one you look at most of the time, and to occasionally use the checkbox to switch screens when you need to take a quick look at what's on the bottom screen.
I wasn't saying this should be the default, just an option that you can turn on or off. Or at least make it so that you can still swap screens when you apply a custom layout in the config ini.
